internal/action/import.go

```diff
@@ -265,6 +309,10 @@ func ImportWorkspaceResources(ctx context.Context, client *tfe.Client, tf *tfexe
 		}
 	}
 
+	if err := ImportRelatedRunTriggers(ctx, tf, client, workspace); err != nil {
```

Comment:
Looks like the API is getting called multiple times to load the triggers. It would be better to load the data once and then pass it around as needed. May be an issue for other resources and therefore not one to be solved in this PR.

Reply:
yep looks like this is a byproduct of adopting drift detection as the primary source for importing without a full refactor of the existing import tools. I updated the run trigger portion since its all new code, but indeed the other resources could use some love